docs/api/components/schemas.yaml

```
@@ -963,6 +963,29 @@ Replication-status:
   required:
     - replication_id
   title: Replication-status
+Scopes:
+  description: A map of all the collections with their corrosponding configs for this scope
+  type: object
+  properties:
+    collection_name:
+      $ref: '#/CollectionConfig'
+  title: Scopes
+CollectionConfig:
+  description: The configuration for the individual collection
+  type: object
+  properties:
+    SyncFn:
+      description: The Javascript function that newly created documents in this collection are ran through.
+      type: string
+      default: 'function(doc){channel(doc.channels);}'
```

This won't actually be the default sync function for a collection in its final implementation. Just keep it as an example.

```suggestion
      example: 'function(doc){channel(doc.channels);}'
```